idprovider/local/cache.go

```diff
@@ -104,13 +104,13 @@ func (c *contactsCache) Write(alias string, p perun.Peer) error {
 	return nil
 }
 
-// Delete deletes the peer from contacts cache.
+// Delete deletes the peer from idProvider cache.
 // Returns an error if peer corresponding to given alias is not found.
-func (c *contactsCache) Delete(alias string) error {
+func (c *idProviderCache) Delete(alias string) error {
 	c.mutex.Lock()
 	defer c.mutex.Unlock()
 	if _, ok := c.peersByAlias[alias]; !ok {
-		return errors.New("peer not found in contacts")
+		return errors.New("peer not found in idProvider")
```

Comment:
```suggestion
		return errors.New("peer not found in ID provider")
```

same as above 
Using ID provider as two words is more readable in the context of comments and documentation.

Reply:
[Okay](https://github.com/hyperledger-labs/perun-node/pull/135/commits/c55f1f275ed66aae884d2380810cc6c5c06190e1)